src/Core.hs

```diff
@@ -97,11 +105,10 @@ processOps core = go Null
       case maybeOp of
         Just op -> do
           let newValue = handleOp op val
-          atomically $ writeTVar (coreCurrentValue core) newValue
-          atomically $ writeTBQueue (coreUpdates core) (Just $ Updated (opPath op) newValue)
+          atomically $ do
+            writeTVar (coreCurrentValue core) newValue
+            writeTBQueue (coreUpdates core) (Just $ Updated (opPath op) newValue)
           go newValue
         Nothing -> do
-          -- Stop the loop when we receive a Nothing. Tell the update loop to
-          -- quit as well.
-          atomically $ writeTBQueue (coreUpdates core) Nothing
```

Comment:
Did you mean to delete this line? How will it signal a `stop` in the update and log queues? Or, how was this useful before and no longer is?